example/main.go

```diff
@@ -31,14 +35,47 @@ func main() {
 		log.Fatalf("Failed to create ocagent-exporter: %v", err)
 	}
 	trace.RegisterExporter(oce)
+	view.RegisterExporter(oce)
+
 	trace.ApplyConfig(trace.Config{
 		DefaultSampler: trace.AlwaysSample(),
 	})
 
+	// Some stats
+	view.SetReportingPeriod(61 * time.Second)
```

Comment:
It seems the same code as the other PR except for the period being 61 here, 4 on the Prometheus one. Why the different values in each case?

Reply:
@pjanotti, Stackdriver requires a minimum of 60 seconds when reporting metrics.